src/AddTodoForm.js

```diff
@@ -1,45 +1,46 @@
 import React, { useState } from 'react';
 
 /* a functional component that takes in props */
-function AddTodoForm(props) {
+function AddTodoForm({ onAddTodo }) {
+  /* the initial state of todoTitle and the function that sets its' value */
   const [todoTitle, setTodoTitle] = useState('');
   
+  /* 
+  a handler function that takes in an onChange event gets event target value
+  as a variable and then calls setTodoTitle function with event target value 
+  variable as an argument
+  */
   function handleTitleChange(event) {
-    let newTodoTitle = event.target.value
+    let newTodoTitle = event.target.value;
```

Comment:
did you find out when to use the semicolon (;)?
>>> Use semi-colons at the end of return statements: This is especially helpful within the context of JS frameworks like react where a return statement can take up the whole function. It helps spot the difference between when the function logic ends and when the actual function ends.

Reply:
I will use semicolon only at the end of a statement from now on. It is a hold over from writing JS code with frequent use of semicolon. In some React code I've seen I had seen frequent semicolon use also. For example after import files at or exporting the default component.